app/jobs/tag_count_update_job.rb

```diff
@@ -0,0 +1,27 @@
+class TagCountUpdateJob < RedisSetJob
+  queue_as :utilities
```

Comment:
Do you want to queue this as utilities? I made the stat counter PR use a separate queue, but this job happens every 2 minutes anyway, so it's not like it's going to swamp everything every half-an-hour. (It might swamp a little if the job gets backed up, though.)

Reply:
I'm inclined to move it to a different queue, but let's see how @zz9pzza and @redsummernight feel.

Reply:
Yeah I think different queues that can be throttled differently are good.

Reply:
Perfect world would be config configurable :) but yes different queues

Reply:
Unfortunately, one of the jobs involved has its queue set in `config/resque_schedule.yml`, so we can't set that queue with a config variable.